CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,11 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.34.0...master)
+
+
+## FxA Client
+
+### Breaking Changes
+
+- The `FirefoxAccount.beginOAuthFlow` method does not require the `wantsKeys` argument anymore
+  as it will always require scopped keys from now on.
```

Comment:
nit: "will always require scoped keys" doesn't sound accurate to me (it's always *capable* of receiving them, but doesn't *require* them) and is probably more detail than consumers need anyway. It might be simpler to say something generic like "and will do the right thing based on the requested scopes".